src/components/Timer/index.js

```diff
@@ -2,21 +2,21 @@ import React from 'react';
 import './index.scss';
 
 const Timer = ({ time }) => {
-    //Double Digit Timers
-    const checkTime = time => {
-        let temp = time.toString();
-        return (temp.length === 2) ? temp : "0" + temp; 
-    }
+	//Double Digit Timers
+	const checkTime = (time) => {
+		let temp = time.toString();
+		return temp.length === 2 ? temp : '0' + temp;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SamooMei/Pomall/pullRequest?prid=7386623)
```suggestion
		return temp.length === 2 ? temp : "0" + temp;
```